editors/code/package.json

```diff
@@ -268,6 +268,125 @@
                 },
                 "pattern": "$rustc"
             }
+        ],
+        "colors": [
+            {
+                "id": "ralsp.background",
+                "description": "Background color",
+                "defaults": {
+                    "dark": "#3F3F3F",
+                    "light": "#001080",
+                    "highContrast": "#000000"
+                }
+            },
+            {
+                "id": "ralsp.comment",
+                "description": "Color for comments",
+                "defaults": {
+                    "dark": "#7F9F7F",
+                    "light": "#008000",
+                    "highContrast": "#7CA668"
+                }
+            },
+            {
+                "id": "ralsp.string",
+                "description": "Color for strings",
+                "defaults": {
+                    "dark": "#CC9393",
+                    "light": "#A31515",
+                    "highContrast": "#CE9178"
+                }
+            },
+            {
+                "id": "ralsp.unsafe",
+                "description": "Color for unsafe",
+                "defaults": {
+                    "dark": "#FF3030",
+                    "light": "#FF1010",
+                    "highContrast": "#FF1010"
+                }
+            },
+            {
+                "id": "ralsp.keyword",
+                "description": "Color for keywords",
+                "defaults": {
+                    "dark": "#F0DFAF",
```

Comment:
FWIW, I'd be perfectly fine with picking a color from default dark theme. This is now configurable, so I can just specify zenburn colors in my config

Reply:
I switched to the default colors.